ivy/functional/backends/paddle/random.py

```diff
@@ -142,7 +142,7 @@ def randint(
 
 def seed(*, seed_value: int = 0) -> None:
```

Comment:
I think this should be a void function? Maybe @vedpatwardhan can chime in here as looks like this is a systematic error (which I'm not sure what are the consequences of removing it)